base/system/format/format.c

```diff
@@ -487,9 +488,11 @@ int wmain(int argc, WCHAR *argv[])
         }
     }
 
-    if (QueryDeviceInformation(RootDirectory,
-                               &DeviceInformation,
-                               sizeof(DeviceInformation)))
+    if (!QueryDeviceInformation(RootDirectory, &DeviceInformation, sizeof(DeviceInformation)))
+    {
+        totalNumberOfBytes.QuadPart = 0;
+    }
+    else
```

Comment:
Alternatively, with less diff (keeping the existing code),
```suggestion
    totalNumberOfBytes.QuadPart = 0;
    if (QueryDeviceInformation(RootDirectory,
                               &DeviceInformation,
                               sizeof(DeviceInformation)))
```

Reply:
Yeah, though I think my proposal is more explicit,
so I would rather keep it, unless you insist..